website/src/components/Survey/LabelInputGroup.tsx

```diff
@@ -0,0 +1,198 @@
+import { Box, Flex, Grid, Spacer, Text, useColorModeValue, VStack } from "@chakra-ui/react";
+import React from "react";
+import { useState } from "react";
+import { TbChevronLeft, TbChevronRight, TbChevronsLeft, TbChevronsRight } from "react-icons/tb";
+import { LikertButtons } from "src/components/Buttons/LikertButtons";
+import { Explain } from "src/components/Explain";
+
+interface LabelInputGroupProps {
+  labelIDs: Array<string>;
+  simple?: boolean;
+  onChange: (values: number[]) => unknown;
+  isEditable?: boolean;
+}
+
+interface LabelInfo {
+  zeroText: string;
+  oneText: string;
+  zeroDescription: string[];
+  oneDescription: string[];
+  inverted: boolean;
+}
+
+// This should be moved to the valid labels api endpoint
```

Comment:
I wonder how annoying it would be to localize this.

Reply:
To localise it where it is now, or once it is moved to the API?

Reply:
oof, we will see...